### ISSUE TITLE
I am facing letter overlapping and got inner html error. I am sharing the screenshot.

### ISSUE THREAD

**Issue by amymal-origami** [peterhry/CircleType#129]
![image_2020_02_03T11_26_09_530Z](https://user-images.githubusercontent.com/15048449/73649917-ada2df00-46a2-11ea-93c9-2a836eb0a9b9.png)

**Comment by muzamilijaz** [comment]
The letters also overlapped on top of each other and won't shows the text in circle !!

**Comment by amymal-origami** [comment]
yup @muzamilijaz is right I am also facing this issue of letter overlapping. Kindly help out in both bugs !

**Comment by StephanWagner** [comment]
Same here! Seems to be font dependent. I'm using https://blambot.com/products/anime-ace-2 and it seems to have trouble with (at least) the letters O and P. Works fine with the system font. Well not really fine, kinda looks to me the letter-spacing in the bottom line is too large (see screenshot). No errors in console.

My setup is quite simple:

```javascript
const circleTypeTop = new CircleType(app.gameScoreFinalTextTop);
circleTypeTop.radius(260);

const circleTypeBottom = new CircleType(app.gameScoreFinalTextBottom);
circleTypeBottom.radius(260).dir(-1);
```

<img width="701" alt="Screenshot 2020-04-24 at 09 43 44" src="https://user-images.githubusercontent.com/1636873/80187807-9673ed80-8610-11ea-814d-032dfdfcffa2.png">

**Comment by StephanWagner** [comment]
I found the problem and posted a pull request: https://github.com/peterhry/CircleType/pull/130

You can also fix this by giving all children `span` elements of the container element the style `display: inline-block`:

```html
<div class="my-text">This is my rotated Text</div>
```

```css
.my-text span {
  display: inline-block;
}
```

At least that worked in my case.

**Comment by peterhry** [comment]
Fixed by pull #130
